experimenter/experimenter/nimbus_ui_new/static/.prettierrc

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
I dont think we need .prettierrc if its empty

Reply:
🤷‍♀️ Handy in case we want to add more rules later?  I expect we'll want to start filling it in as soon as we start adding more little scripts and things.